### DISCUSSION TITLE
Refund for Google Maps Scraper Pro

### DISCUSSION THREAD

**Discussion by Chetan11-dev** [omkarcloud/google-maps-scraper#44] [General]
To request a refund for the Google Maps Scraper Pro Version, follow the instructions in the README [here](https://github.com/omkarcloud/google-maps-scraper#-how-do-i-get-a-refund), and then, leave a comment with the following template: 

```
I have sent a refund request using my email: myname@example.com.
```

**Comment by ryolay** [comment]
Please initiate a refund. My communication email is c.dslvp@gmail.com(mailto:c.dslvp@gmail.com)

**Reply by Chetan11-dev** [reply]
Kindly check your Email.

**Reply by Chetan11-dev** [reply]
Your refund has been successfully issued!

**Reply by cmurr05** [reply]
I have requested a refund at cadenmurray205@gmail.com

**Comment by hunkardoner** [comment]
I have sent a refund request from my email: hunkardoner@gmail.com

**Reply by Chetan11-dev** [reply]
Your refund has been successfully issued!

**Comment by Aoyamaxx** [comment]
I have sent a refund request from my email: quanpu.xiao@gmail.com

**Reply by Chetan11-dev** [reply]
Your refund has been successfully issued!

**Comment by thotranhuu99** [comment]
I have sent a refund request using my email: thotranhuu99@gmail.com

**Reply by Chetan11-dev** [reply]
Your refund has been successfully issued!

**Comment by JMLeblanc23** [comment]
I have sent a refund request from my email: jefffleblanc@yahoo.com

**Reply by Chetan11-dev** [reply]
Your refund has been successfully issued!

**Comment by pablos1rvent** [comment]


**Reply by Chetan11-dev** [reply]
Your refund has been successfully issued!

**Comment by Fluppe1** [comment]
I have sent a refund request from my email: kevinhoffmann2000@gmail.com

**Reply by Chetan11-dev** [reply]
Your refund has been successfully issued!

**Comment by jonatanaerial** [comment]
I have sent a refund request from my email: jonatan.sanchez@aerialfilms.es

**Reply by Chetan11-dev** [reply]
Your refund has been successfully issued!

**Comment by lovekurdt** [comment]
I have sent a refund request from my email: [lovekurdt@gmail.com](mailto:lovekurdt@gmail.com)

**Reply by Chetan11-dev** [reply]
Your refund has been successfully issued!

**Comment by Louis554** [comment]
I have sent a refund request from my email: louis.maillot2003@gmail.com

**Reply by Chetan11-dev** [reply]
Your refund has been successfully issued!

**Comment by priyanshu2510** [comment]


**Reply by Chetan11-dev** [reply]
Your refund has been successfully issued!

**Comment by Dvalon** [comment]
I have sent a refund request from my email: dylan.wt@outlook.com

**Reply by Chetan11-dev** [reply]
Your refund has been successfully issued!

**Comment by Armstock** [comment]
Please initiate a refund. My communication email is [ArmstockLiu@gmail.com](mailto:ArmstockLiu@gmail.com)

**Reply by Chetan11-dev** [reply]
Your refund has been successfully issued!

**Comment by tapriliando** [comment]
> I have sent a refund request using my email: triesapriliando@gmail.com
> ```

**Reply by Chetan11-dev** [reply]
Your refund has been successfully issued!

**Comment by tonySwiftDev** [comment]
I have sent a refund request using my email: 549047626@qq.com

**Reply by Chetan11-dev** [reply]
Your refund has been successfully issued!

**Comment by njain006** [comment]
I have sent a refund request from my email: nitinj025@gmail.com

**Reply by Chetan11-dev** [reply]
Your refund has been successfully issued!

**Comment by aaygupta** [comment]
I have sent a refund request using my email: aaygupta@outlook.com

**Reply by Chetan11-dev** [reply]
Hi Ankit, Kindly check your mail.

**Reply by Chetan11-dev** [reply]
Your refund has been successfully issued!

**Comment by rayroman1160** [comment]
I have sent a refund request using my email: rayroman1160@gmail.com.

**Reply by Chetan11-dev** [reply]
Hi Ray, Kindly check your mail.

**Reply by Chetan11-dev** [reply]
Your refund has been successfully issued!

**Comment by cmurr05** [comment]
I have requested a refund at [cadenmurray205@gmail.com](mailto:cadenmurray205@gmail.com)

**Reply by Chetan11-dev** [reply]
Your refund has been successfully issued!

**Comment by zeroazulagencia** [comment]
I have sent a refund request from my email: cristian.parada@zeroazul.com

**Reply by Chetan11-dev** [reply]
Your refund has been successfully issued!

**Comment by pandadezzz** [comment]
I have sent a refund request from my email: cheno1201@gmail.com.

**Reply by Chetan11-dev** [reply]
Your refund has been successfully issued!

**Comment by huayra207** [comment]
I have sent a refund request using my email: jadriel207@gmail.com

**Reply by Chetan11-dev** [reply]
Your refund has been successfully issued!

**Comment by sbdmarketing** [comment]
I have sent a refund request using my email: jamestheboss71699@gmail.com.

**Reply by Chetan11-dev** [reply]
Your refund has been successfully issued!

**Comment by Snichoo** [comment]
I have sent a refund request from my email: sam.satoshi.nichols1223@gmail.com

**Reply by Chetan11-dev** [reply]
Your refund has been successfully issued!

**Comment by mbg1996** [comment]
I have sent a refund request using my email: maxbgleit@gmail.com

**Reply by Chetan11-dev** [reply]
Your refund has been successfully issued!

**Comment by eyjafjallajoskull** [comment]


**Reply by Chetan11-dev** [reply]
Your refund has been successfully issued!

**Comment by Cileungsi333** [comment]
I have sent a refund request using my email: jakartaastraisuzu@gmail.com

**Reply by Chetan11-dev** [reply]
Your refund has been successfully issued!

**Comment by weikhor** [comment]
I have sent a refund request using my email: khorcheanwei516@gmail.com

**Reply by Chetan11-dev** [reply]
Your refund has been successfully issued!

**Comment by ciocirlieionut** [comment]
I have sent a refund request from my email: ciocirlieionutalexandru@gmail.com

**Reply by Chetan11-dev** [reply]
Hi, Kindly check your mail.

**Reply by Chetan11-dev** [reply]
Your refund has been successfully issued!

**Comment by jedevar** [comment]
I have sent a refund request from my email: jedevar@gmail.com

**Reply by Chetan11-dev** [reply]
Hi, Kindly check your mail.

**Reply by Chetan11-dev** [reply]
Your refund has been successfully issued!

**Comment by AsmanyAkter** [comment]
I have sent a refund request from my email:  asmany2016@gmail.com

**Reply by Chetan11-dev** [reply]
Your refund has been successfully issued!

**Comment by pedrogius** [comment]


**Reply by Chetan11-dev** [reply]
Your refund has been successfully issued!

**Comment by cvv6679** [comment]
I have sent a refund request from my email.  cvv6679@gmail.com

**Reply by Chetan11-dev** [reply]
Your refund has been successfully issued!

**Comment by Anthony2033** [comment]
I have sent a refund request from my email: anthonyc2366@gmail.com

**Reply by Chetan11-dev** [reply]
Your refund has been successfully issued!

**Comment by Arikalen** [comment]
I have sent a refund request from my email: megaprint6@gmail.com

**Reply by Chetan11-dev** [reply]
Your refund has been successfully issued!

**Comment by complex5a** [comment]


**Reply by Chetan11-dev** [reply]
Your refund has been successfully issued!

**Comment by harveyjhuang** [comment]
I have sent a refund request using my email: harvey455135@gmail.com

**Reply by Chetan11-dev** [reply]
Your refund has been successfully issued!

**Comment by joseluisantiguarosa987** [comment]
Cómo están bien

**Comment by Ziakimbogo** [comment]
I have sent a refund request from my email: chaviallegeraud@gmail.com / chaineytbdegreg@gmail.com

**Reply by Chetan11-dev** [reply]
Your refund has been successfully issued!

**Comment by Chetan11-dev** [comment]
Closed Discussions, as refund functionality moved to website. Mail us happy.to.help@omkar.cloud, if you need help.
